src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -764,14 +764,38 @@ default boolean lootIncludeClueScrolls() {
         return true;
     }
 
+    @ConfigItem(
+        keyName = "lootItemAllowlist",
+        name = "Item Allowlist",
+        description = "Always fire notifications for these items, despite value settings. " +
```

Comment:
Should we clarify that items that pass value threshold still dink, even if not on explicit allowlist (assuming not on denylist)